enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -998,10 +1034,19 @@ def get_group_member_data_with_aggregates(self, request, uuid):
         # requester.
         current_url = request.build_absolute_uri()
         if not traverse_pagination:
-            if member_response.get('next'):
-                member_response['next'] = current_url.replace(f"page={page}", f"page={str(int(page) + 1)}")
-            if member_response.get('previous'):
-                member_response['previous'] = current_url.replace(f"page={page}", f"page={str(int(page) - 1)}")
+            # If sorting by enrollment count and a provided page, we have to be more cleaver about constructing next
+            # and previous values
+            if sort_by_enrollment_count:
+                if page_index_start > 0:
+                    member_response['previous'] = current_url.replace(f"page={page}", f"page={str(int(page) - 1)}")
```

Comment:
nit: Shouldn't need to cast to a string inside an f-string expression.  Also, could make this a helper function.
```suggestion
                    member_response['previous'] = current_url.replace(f"page={page}", f"page={int(page) - 1}")
```